src/us/kbase/common/executionengine/SubsequentCallRunner.java

```diff
@@ -141,6 +154,27 @@ public ModuleRunVersion getModuleRunVersion() {
             return jsonRpcResponse;
         }
     }
+    
+    private void logError(String name, Integer code, String message, String data) {
+        cbLogErr("Error is thrown by subjod");
```

Comment:
You might also want to consider making the error a single string so it'd less likely errors from jobs run in parallel are interleaved.